docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -10,6 +10,14 @@ compatibility notes.
 **Python 2.7 support ended on June 1, 2019**. (:issue:`501`)
 **Minimum numpy version is now 1.10.4. Minimum pandas version is now 0.18.1.**
 
+API Changes
+~~~~~~~~~~~
+* Moved functions for cell temperature from `pvsystem.py` to `celltemp.py`, renamed from `pvsystem.sapm_celltemp` to `celltemp.sapm`, and from `pvsystem.pvsyst_celltemp` to `celltemp.pvsyst`. Argument order in `celltemp.sapm` changed and output type from `celltemp.pvsyst` have been changed to be consistent in `celltemp.py` for all cell temperature model functions. (:issue:`678')
+* Implemented `pvsyst` as a key value for `ModelChain.temp_model`. (:issue:`678')
```

Comment:
nit: I think "as an option" is more clear than "as a key value". (key value makes me think key value pair which isn't quite right here.)